group_vars/development

```diff
@@ -1,12 +1,13 @@
 mysql_root_password: devpw
 
+deploy_user: vagrant
+
 wordpress_sites:
-  - site_name: example.dev
+  example.dev:
     site_hosts:
       - example.dev
+    repo: https://github.com/roots/bedrock.git
```

Comment:
Sorry @mAAdhaTTah  I didn't read your comment carefully. I had thought `repo` was already missing from development and you were asking why. I was wrong about what you were saying, and wrong about what was in the code.